parcllabs/parcllabs_client.py

```diff
@@ -9,19 +9,31 @@
 from parcllabs.services.search import SearchMarkets
 
 
-class ParclLabsClient:
-    def __init__(self, api_key: str, limit: int = 12):
-        """
-        Create a ParclLabsClient client.
+class Endpoint:
+    def __init__(self, client, url, limit):
+        self.client = client
+        self.url = url
+        self.limit = limit
```

Comment:
Is this class necessary? I don't see it being used anywhere.

Reply:
Removed, great catch!